langchain/chains/openai_functions/citation_fuzzy_match.py

```diff
@@ -0,0 +1,82 @@
+from typing import List
+from pydantic import Field, BaseModel
+
+from langchain.base_language import BaseLanguageModel
+from langchain.output_parsers.openai_functions import (
+    PydanticOutputFunctionsParser,
+)
+from langchain.chains.llm import LLMChain
+from langchain.prompts.chat import ChatPromptTemplate, HumanMessagePromptTemplate
+from langchain.schema import SystemMessage, HumanMessage
+
+
+class Fact(BaseModel):
```

Comment:
FactWithProvenance? FactWithEvidence?

The model being named Fact and having an attribute called `fact` feels isn't a good representation of the information